pkg/operator/helpers_test.go

```diff
@@ -0,0 +1,44 @@
+package operator
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"testing"
+
+	arov1alpha1 "github.com/Azure/ARO-RP/pkg/operator/apis/aro.openshift.io/v1alpha1"
+	_ "github.com/Azure/ARO-RP/pkg/util/scheme"
+)
+
+func aroCluster(domains []string) *arov1alpha1.Cluster {
+	return &arov1alpha1.Cluster{
+		Spec: arov1alpha1.ClusterSpec{
+			GatewayDomains: domains,
+		},
+	}
+}
+
+func TestGatewayEnabled(t *testing.T) {
+	tests := []struct {
+		name        string
+		cluster     *arov1alpha1.Cluster
+		wantEnabled bool
+	}{
+		{
+			name:    "gateway disabled",
+			cluster: aroCluster([]string{}),
+		},
```

Comment:
Go convention.  

if you don't define a value, it defaults to the zero-value for the type, which for booleans is `false`.

Reply:
I get that it will do that but not stating it explicitly feels... wrong :smile: Maybe my eyes have become a bit too pythonic :wink: 

Do you have any nice Go conventions/style guide/blog/something like that you could recommend?

Reply:
@mociarain In case it is useful, those are some good references (IMHO):
- [The Google Go style guide](https://google.github.io/styleguide/go/)
- [Effective Go](https://go.dev/doc/effective_go)
- [Practical Go: Real world advice for writing maintainable Go programs](https://dave.cheney.net/practical-go/presentations/qcon-china.html)
- [Uber Go style guides](https://github.com/uber-go/guide/blob/master/style.md)

Rob Pike and [Dave Cheney](https://dave.cheney.net/practical-go) are my 2 go to reference about Go.
🙂